app/src/main/java/com/example/android/architecture/blueprints/todoapp/di/FragmentScoped.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.android.architecture.blueprints.todoapp.di
+
+
+import javax.inject.Scope
+import kotlin.annotation.AnnotationRetention.RUNTIME
+
+@Scope
+@Retention(RUNTIME)
+@Target(
+    AnnotationTarget.CLASS, AnnotationTarget.FILE, AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER
+)
+annotation class FragmentScoped
```

Comment:
I don't know if this annotation does anything because it's not related to a Dagger `Component`

Reply:
I think dagger android let's you add custom scope annotations to then contributes android injector builders which then go on the auto generated components. Only 70% sure

Reply:
Oh, that makes sense :)

Reply:
Removed scopes